scripts/size_compare.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+#
+# This scripts comparse local file sizes with remote files and print an overview
+#
+
+import os
+import subprocess
+import multiprocessing
+from urllib import request
+
+# where to look for the local files
+target_dir = "./bin" + os.environ.get("TARGET_DIR", "")
+
+# where to look for the remote files
+base_url = os.environ.get("BASE_URL", "https://downloads.openwrt.org/snapshots")
```

Comment:
You should make clear in the commit message what exactly "remote" is, i.e. that it's "https://downloads.openwrt.org/snapshots" (at least by default).

Despite, this looks like BASE_URL can also be used as parameter? Then you should also add that.